v2/json2/server.go

```diff
@@ -7,12 +7,13 @@ package json2
 
 import (
 	"encoding/json"
+	"log"
 	"net/http"
 
 	"github.com/gorilla/rpc/v2"
 )
 
-var null = json.RawMessage([]byte("null"))
+// var null = json.RawMessage([]byte("null"))
```

Comment:
Can you give some context on this? Is it not being used anywhere?

Reply:
Yes, that type is nowhere used in v2/json2/server.go.
Some examples of it being used were found specifically in [WriteResponse](https://github.com/gorilla/rpc/blob/main/v2/protorpc/server.go#L119-L123) where it uses the datatype for Error, but, specifically for v2/json2/server.go, the error key hasn't been specified in [WriteResponse](https://github.com/gorilla/rpc/blob/main/v2/json2/server.go#L179-L183). I am not sure of the exact reason yet, but as it wasn't used anywhere, golangci-lint complained about it, so, I've commented it for the time being.

Reply:
Sounds good.